.github/workflows/test-docker-container.yml

```diff
@@ -0,0 +1,22 @@
+name: Build and Run Docker Container
+
+on:
+  pull_request:
+  push:
+    branches: main
+
+jobs:
+  build_and_run:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Build Docker image
+        run: |
+          docker build -t zallet .
+
+      - name: Run command inside Docker container
+        run: |
+          docker run --rm zallet -h
```

Comment:
FYI the problem you last commit was fixing is that you tried to use `-help` as the flag, which is incorrect for Zallet. `--help` is what you should use (or `-h`), as the help text itself should show.

In general, `zcashd`'s support for single-hypen word flags to CLI is not something we are going to support in Zallet.